make/langtools/src/classes/build/tools/symbolgenerator/CreateSymbols.java

```diff
@@ -2384,7 +2384,10 @@ private boolean readAttribute(ClassFile cf, FeatureDescription feature, Attribut
                 MethodDescription method = (MethodDescription) feature;
                 method.methodParameters = new ArrayList<>();
                 for (MethodParameters_attribute.Entry e : params.method_parameter_table) {
-                    String name = cf.constant_pool.getUTF8Value(e.name_index);
+                    String name = null;
+                    if (e.name_index != 0) {
```

Comment:
could use ternary operator here

Reply:
`name_index` could always have been 0 to indicate an absent parameter name: https://docs.oracle.com/javase/specs/jvms/se20/html/jvms-4.html#jvms-4.7.24

Reply:
I see, then this was a bug in the previous code, still I suggest using the ternary operator here